src/test/java/uk/gov/pay/api/resources/directdebit/DirectDebitPaymentsResourceIT.java

```diff
@@ -0,0 +1,179 @@
+                .withAmount(AMOUNT)
+                .withPaymentProvider(PAYMENT_PROVIDER)
+                .withCreatedDate(CREATED_DATE)
+                .withDescription(DESCRIPTION)
+                .withMandateId(MANDATE_ID)
+                .withProviderId(PROVIDER_ID)
+                .withState(new DirectDebitPaymentState(status, finished, DETAILS_FIELD))
+                .withReference(REFERENCE)
+                .build();
+
+        connectorDDMockClient.respondWithPaymentCreated(connectorResponse, account.getAccountId());
+
+        String request = new Gson().toJson(Map.of(
+                "amount", AMOUNT,
+                "description", DESCRIPTION,
+                "reference", REFERENCE,
+                "mandate_id", MANDATE_ID));
+
+        postPaymentResponse(request)
+                .statusCode(201)
+                .body("payment_id", is(PAYMENT_ID))
+                .body("amount", is((int)AMOUNT))
+                .body("payment_provider", is(PAYMENT_PROVIDER))
+                .body("created_date", is(CREATED_DATE))
+                .body("description", is(DESCRIPTION))
+                .body("reference", is(REFERENCE))
+                .body("state.status", is(status))
+                .body("state.finished", is(finished))
+                .body("_links.events.href", is(paymentEventsLocationFor(PAYMENT_ID)))
+                .body("_links.events.method", is("GET"))
+                .body("_links.self.href", is(paymentLocationFor(PAYMENT_ID)))
+                .body("_links.self.method", is("GET"))
+                .body("_links.mandate.href", is(mandateLocationFor(MANDATE_ID)))
+                .body("_links.mandate.method", is("GET"));
+    }
+
+    @Test
+    public void createPayment_successWhenDescriptionNotPresent() throws JsonProcessingException {
+        final Account account = new Account(GATEWAY_ACCOUNT_ID, TokenPaymentType.DIRECT_DEBIT);
+        final String status = "created";
+        final boolean finished = false;
+
+        DirectDebitConnectorCreatePaymentResponse connectorResponse = aDirectDebitConnectorCreatePaymentResponse()
+                .withPaymentExternalId(PAYMENT_ID)
+                .withAmount(AMOUNT)
+                .withPaymentProvider(PAYMENT_PROVIDER)
+                .withCreatedDate(CREATED_DATE)
+                .withState(new PaymentState(status, finished))
+                .withReference(REFERENCE)
+                .build();
+
+        connectorDDMockClient.respondWithPaymentCreated(connectorResponse, account.getAccountId());
+
+        String request = new Gson().toJson(Map.of(
+                "amount", AMOUNT,
+                "reference", REFERENCE,
+                "mandate_id", MANDATE_ID));
+
+        postPaymentResponse(request)
+                .statusCode(201)
+                .body("$", not(hasKey("description")));
+    }
+
+    @Test
+    @Parameters(method = "parametersForValidation")
+    public void validationFailures(
```

Comment:
The test class was renamed from DirectDebitPaymentsResourceTest and this validationFailures test added. Git wasn't clever enough to recognise the rename